src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,52 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
+    'equivalence': '>A oboInOwl:source',
+    'ORCID': '>A oboInOwl:source SPLIT=|',
+}
+
+
+def sssom_to_robot_template(inpath: Union[str, Path], outpath: Union[str, Path]):
+    """Convert SSSOM to ROBOT template."""
+    msdf: MappingSetDataFrame = parse_sssom_table(inpath)
+    df: pd.DataFrame = msdf.df
+
+    # Filter
+    df = df[df['predicate_id'] == 'skos:exactMatch']
+
+    # Conversion
+    df = df[['subject_id', 'subject_label', 'object_id', 'object_label']].sort_values(['subject_id', 'object_id'])
+    df['equivalence'] = 'MONDO:equivalentTo'
+    df['ORCID'] = '|'.join(msdf.metadata['creator_id'])
+    df = pd.concat([pd.DataFrame([ROBOT_ROW]), df])
+
+    # Write
+    df.to_csv(outpath, sep='\t', index=False)
+
+
+def cli():
```

Comment:
This code should really be in mondolib!

Reply:
@matentzn That was the original goal, and I created https://github.com/monarch-initiative/mondolib/issues/11 in `mondolib` for that reason, but:
1. The current state of `mondolib` doesn't lean itself well to additions; needs a refactor.
2. For now, @twhetzel has OK'ed additional Python in `mondo-ingest`.

But I think we both would agree that eventually it should make its way there.

Reply:
I Ok'd more Python in `mondo-ingest` for the ORDO(?) work needed before the release, but ideally anything new should go in `mondolib`... let me see if I can refactor enough to get a stub for adding new Python code into `mondolib` for the near future.

_IF_ this work were moved to `mondolib` how soon does it need to be in there?

Reply:
Ah, OK I think I heard this without the explicit ORDO qualifier. I'm totally fine w/ you/me/us refactoring this into / starting up work on `mondolib` whenever.

Reply:
@matentzn FYI Trish requested that I this work move this over to `mondolib`. I probably won't complete that until 1-3 weeks from now, though, so we don't need to hold up this PR until that's done; can remove this code in a subsequent PR.

Reply:
There is no great urgency here, as I can (a) work of the branch for now and (b) there are multiple issues to iron out with the raw data!

Reply:
Got it! Spoke with Trish and we prioritized https://github.com/monarch-initiative/mondolib/issues/12 for later.

Reply:
And just to clarify, https://github.com/monarch-initiative/mondolib/issues/12 was prioritized for later since Joe was pulled into work on subsets (e.g. all the ORDO PRs and this PR).